content/en/docs/Installation/_index.md

```diff
@@ -82,9 +137,30 @@ kubectl create -f https://raw.githubusercontent.com/kyverno/kyverno/main/config/
 You can also pull from a release branch to install the stable releases including release candidates.
 
 ```sh
-kubectl create -f https://raw.githubusercontent.com/kyverno/kyverno/release-1.6/config/release/install.yaml
+kubectl create -f https://raw.githubusercontent.com/kyverno/kyverno/release-1.7/config/release/install.yaml
 ```
 
+## Security vs Operability
+
+Regardless of the installation method used for Kyverno, it is important to understand the risks associated with any webhook and how it may impact cluster operations and security especially in production environments. Kyverno configures its resource webhooks by default (but [configurable](/docs/writing-policies/policy-settings/)) in [fail closed mode](https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/#failure-policy). This means if the API server cannot reach Kyverno in its attempt to send an AdmissionReview request for a resource that matches a policy, the request will fail. For example, a validation policy exists which checks that all Pods must run as non-root. A new Pod creation request is submitted to the API server and the API server cannot reach Kyverno. Because the policy cannot be evaluated, the request to create the Pod will fail. Care must therefore be taken to ensure that Kyverno is always available or else configured appropriately to exclude certain key Namespaces, specifically that of Kyverno's, to ensure it can receive those API requests. There is a tradeoff between security by default and operability regardless of which option is chosen.
+
+The following combination may result in cluster inoperability if the Kyverno Namespace is not excluded:
+
+1. Kyverno is configured in fail closed mode (default).
+2. No Namespace exclusions have been configured for at least the Kyverno Namespace, possibly other key system Namespaces (ex., `kube-system`).
+3. All Kyverno Pods become unavailable due to a full cluster outage or improper scaling in of Nodes (for example, a cloud PaaS destroying too many Nodes in a node group as part of an auto-scaling operation without first cordoning and draining Pods).
+
+If this combination of events occurs, the only way to recover is to manually delete the ValidatingWebhookConfigurations thereby allowing new Kyverno Pods to start up. Recovery steps are provided in the [troubleshooting section](/docs/troubleshooting/#api-server-is-blocked).
```

Comment:
Should we also point out that kubernetes doesn't allow setting policies on validation/mutation webhook configs to allow for recovery.

Reply:
Are you talking about the ability for a policy to validate a `validatingwebhookconfiguration` resource, for example?

Reply:
Yup :)

Reply:
Does adding the following note beneath this line provide enough clarification?

> Kubernetes does not permit the checking of webhook resources by admission controllers.

Reply:
It does.